src/activeLLM/suggestFix.js

```diff
@@ -9,7 +9,19 @@ export async function suggestFix(
     setState,
 ) {
 
-    const prompt = `Here is a design rule and its description: ${rule} 
+    const prompt = `Here is a design rule and its description: ${rule}
+    Here is a code example that follows this design rule: ${example}
+    The example file path is ${exampleFilePath}
+    Now, here is a code snippet that violates this design rule: ${violation}
+    The violated code's file path is ${violationFilePath}
+    Can you suggest a fix to make this violation follow the given design rule?
+    Generate code with surrounding code included that follows the design rule.
+    Be sure to maintain proper whitespace with \\t and \\n.
+    Give a brief explanation of your fix as well.
+    Ensure that you include the fileName where the fix should be inserted at in the format Example.java.
+    Strictly output in JSON format. The JSON should have the following format:{"code": "...", "explanation": "...", "fileName": "..."}`;
+
+    /*const prompt = `Here is a design rule and its description: ${rule} 
```

Comment:
We may forget why we have this commented prompt. Either add a comment describing it. Or even better remove it if we are not going to use it.